vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -506,6 +506,19 @@ function Test-WindowsDefenderPlatformUpdate {
     }
 }
 
+function Test-ToolsToCacheOnVHD {
+    $toolsDir = "c:\tools"
```

Comment:
Could you also help to update vhdbuilder/packer/test/windows-files-check.ps1 to validate the signature of the new binaries?

Reply:
Confirmed, it will validate